samples/charts/financial-chart/annotations/Services/StockUtility.cs

```diff
@@ -0,0 +1,193 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Samples.Services
+{
+    
+
+class StocksUtility {
```

Comment:
indent class and its content

Reply:
this class should be `public static` and all its fields/functions should be `public static`